src/translations/en-US.json

```diff
@@ -300,18 +300,23 @@
       "limit": 0
    },
    "visualizations.menu.aggregations": {
-      "value": "Aggregations",
-      "comment": "",
-      "limit": 0
+      "value": "Aggregate",
+      "comment": "Heading showed above menu listing possible aggregations (totals).",
+      "limit": 50
    },
    "visualizations.menu.aggregations.rows": {
       "value": "Rows",
-      "comment": "Rows of a table",
+      "comment": "Heading showed above row attributes of the table you may apply one of aggregation (total) on.",
       "limit": 25
    },
    "visualizations.menu.aggregations.all-rows": {
-     "value": "All rows",
-     "comment": "All rows of a table",
+     "value": "of all rows",
+     "comment": "Example: Sum (total) 'of all rows' will be showed in the table",
+     "limit": 25
+   },
+   "visualizations.menu.aggregations.within": {
+     "value": "within",
+     "comment": "Example: Sum (total) 'within' specified attribute will be showed in the table",
```

Comment:
I don't think that this and previous example will tell something to the translator. This will get better with parametrization.

Reply:
Okay, I will figure out sth else